softdev.bib

```diff
@@ -2797,3 +2797,13 @@ @misc{v8
     year = {2008},
     author = {Google},
 }
+
+@misc{rebench18marr,
```

Comment:
I think this file is mostly 4-space indentation? If so, can we match that?

Reply:
Sure,  fixed: f3963a892cbd2177a05d223e6f41a0f0461c6914